bin/hdfcoinc/pycbc_coinc_statmap

```diff
@@ -146,7 +205,142 @@ else:
     f['foreground/ifar_exc'] = numpy.array([])
     f['foreground/fap_exc'] = numpy.array([])
 
-if 'name' in d.attrs:
-    f.attrs['name'] = d.attrs['name']
+if 'name' in all_trigs.attrs:
+    f.attrs['name'] = all_trigs.attrs['name']
+
+# Incorporate hierarchical removal for any other loud triggers
+logging.info("Beginning hierarchical removal of foreground triggers.")
+
+# Step 1: Create arrays to use for reference when storing ifar and fap of
+#         foreground triggers that get hierarchically removed.
+
+# Set an index to keep track of how many hierarchical removals we want to do.
+h_iterations = 0
+
+save_these_fore_ind = numpy.array([], dtype=int)
+orig_fore_stat = fore_stat
```

Comment:
See later comment, but this isn't needed, and I think is not used correctly in any case.